ctapipe/io/dl1eventsource.py

```diff
@@ -0,0 +1,389 @@
+from ctapipe.io.datalevels import DataLevel
+from ctapipe.instrument import (
+    TelescopeDescription,
+    SubarrayDescription,
+    CameraGeometry,
+    OpticsDescription,
+)
+from ctapipe.containers import EventAndMonDataContainer, EventType
+from ctapipe.containers import MCHeaderContainer, MCEventContainer
+from ctapipe.io.eventsource import EventSource
+from astropy.coordinates import Angle
+import astropy.units as u
+import tables
+import warnings
+
+
+class DL1EventSource(EventSource):
+    def __init__(
+        self,
+        input_url,
+        config=None,
+        parent=None,
+        **kwargs
+    ):
+        """
+        EventSource for dl1 files in the standard DL1 data format
+
+        Parameters:
+        -----------
+        input_url : str
+            Path of the file to load
+        config: ??
+        parent: ??
+        kwargs
+        """
+        super().__init__(
+            input_url=input_url,
+            config=config,
+            parent=parent,
+            **kwargs
+        )
+
+        self.file_ = tables.open_file(input_url)
+        self._subarray_info = self._prepare_subarray_info(
+            self.file_.root.configuration.instrument
+        )
+        self._mc_header = self._parse_mc_header(
```

Comment:
This should be optional, we shoudl support observed dl1 files out of the box here

Reply:
so something like `if 'simulation' in self.file_.root.configuration`

Reply:
True. Could just let the parsing function return None if no mcheader table is included?

Reply:
Not as long as you directly give it the `simulation` group here, this will raise an AttributeError or Keyerror before you enter the function

Reply:
Same for the event wise information